src/main/java/net/neoforged/neoforge/event/OnDatapackSyncEvent.java

```diff
@@ -32,6 +33,14 @@ public PlayerList getPlayerList() {
         return this.playerList;
     }
 
+    /**
+     * @return A stream of players to sync datapacks to. If a player is specified,
+     *         the stream will only contain that player.
+     */
```

Comment:
```
    /**
     * Creates a stream of players that need to receive data during this event, which is the specified player (if present) or all players.
     * 
     * @return A stream of players to sync data to.
     */
```

Reply:
To clarify: using only `@return` is bad practice because the method summary is then empty in the html-rendered javadoc.

Reply:
Correct; but that can be remedied with use of the inline return `{@return ...}` tag.

I proposed this rewrite to separate functionality (creation of a new `Stream`) and intent (the specific contents of the stream) from the definition of the return value (a stream of players).

Reply:
Should I update the other methods in this class so that the summary is correctly populated for those? @Technici4n

Reply:
As you prefer:)

Reply:
I have updated the summaries for all the methods in the class.